go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/containerd/containerd
 go 1.16
 
 require (
-	github.com/Microsoft/go-winio v0.4.17
+	github.com/Microsoft/go-winio v0.5.0
```

Comment:
IIRC, there were some breaking changes in v0.5.0, so we may have to be careful with this one

Reply:
For reference; https://github.com/microsoft/go-winio/pull/197#issuecomment-795676667